assets/src/edit-story/components/panels/design/layer/layer.js

```diff
@@ -70,15 +72,13 @@ const LayerIconWrapper = styled.div`
   flex-shrink: 0;
   display: flex;
   align-items: center;
-  justify-content: center;
-  margin-left: 8px;
-  margin-right: 12px;
+  justify-content: flex-start;
   color: ${({ theme }) => theme.colors.fg.primary};
 `;
 
 const LayerDescription = styled.div`
   position: relative;
-  width: calc(100% - 60px);
+  width: 100%;
```

Comment:
To miina's point, this will fix the overflow issues

```suggestion
overflow: hidden;
  width: 100%;
```

Reply:
Perfecto
![image](https://user-images.githubusercontent.com/22185279/114062581-e913dd00-9854-11eb-8941-ec0544ee6b33.png)